js/Main.js

```diff
@@ -0,0 +1,177 @@
+"use strict";
+
+let roundCounter = 0,
+		p1MaxHP = 666,
+		p2MaxHP = 666,
+		interval;
+
+//import Robots object, and jquery
+let Robots = require('./Robot.js'),
+		$ = require('jquery'),
+
+//create player1 and 2
```

Comment:
Watch your indentation here. Hard to tell at first that lines 13 and 14 are part of the let statement from line 9